packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/constants.ts

```diff
@@ -0,0 +1,26 @@
+export const TOKENS = {
```

Comment:
nit: Not sure if we have a rule for that but most of enums I see in the codebase do not use uppercase notation, rather they are like this:

```
enum BulkBalancesErrors {
  InvalidToken = 'Invalid token',
}
```

Reply:
Hm, should i update the one's i added to be like this?

Reply:
up to you